graviti/utility/requests.py

```diff
@@ -219,3 +220,30 @@ def read(self, amt: Optional[int] = None) -> bytes:
 
         except StopIteration:
             return b""
+
+
+_T = TypeVar("_T")
+
+
+def multithread_upload(
```

Comment:
This function can be used in any case not only upload, shall we upload the function name?